les03_task03.py

```diff
@@ -0,0 +1,28 @@
+"""
+В массиве случайных целых чисел поменять местами минимальный и максимальный элементы.
+"""
+
+import random
+
+lst = [random.randint(1, 10) for el in range(6)]
+print('Генерируем массив случайных чисел')
+print(lst)
+max = lst[0]

```

Comment:
рассказывал на лекции, что не стоит переопределять встроенные функции.
```suggestion
max_ = lst[0]

```